daemon/api_debug.go

```diff
@@ -63,18 +63,43 @@ func getBaseDeclaration(st *state.State) Response {
 	}, nil)
 }
 
+func checkConnectivity(theStore snapstate.StoreService) Response {
```

Comment:
This looks good.

Reply:
looking a bit more it's a bit unclear why this is not taking the state and getting the store on its own as well